packages/components/src/local/route/helpers/planned-routes-for.ts

```diff
@@ -0,0 +1,56 @@
+import { LatLng } from 'leaflet'
+
+import SergeHex from '../../mapping/types/serge-hex'
+import SergeGrid from '../../mapping/types/serge-grid'
+import hexNamed from '../../hex-grid/helpers/hex-named'
+
+export interface RouteData {
+  /**
+   *  line representing this route (may include multiple steps per turn)
+   */
+  polyline: LatLng[]
+  /** 
+   * series of end-of-turn locations, one per turn
+   */ 
+  turnEnds: LatLng[]
+}
+
+export const LENGTH_OF_TRIMMED_LINE: number = 2
```

Comment:
Yeah, I definitely see no reason for it to use this formatting. It should be `lengthOfTrimmedLine`

Reply:
yes, done.